Blue_Green_Ami.pkr.hcl

```diff
@@ -0,0 +1,89 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.1.1"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+variable "ami_prefix1" {
```

Comment:
Could you provide a proper name for the variable please, this will have allow for a better debug later.